services/catalog/src/simcore_service_catalog/service/catalog_services.py

```diff
@@ -524,7 +528,7 @@ async def list_user_service_release_history(
 
     total_count, history = await repo.get_service_history_page(
         # NOTE: that the service history might be different for each user
-        # since access-rights are defined on a k:v basis
+        # since access rights are defined on a k:v basis
```

Comment:
I do not understand this line of the comment. I believe that is clear that the history changes for each user.